Makefile

```diff
@@ -0,0 +1,13 @@
+GOROOT := $(shell go env GOROOT)
+
+.PHONY: serve
+all: help
```

Comment:
s/help/wasm

or better, move the `wasm` target to the top, remove the `all` target.